notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">I will handle not reuired missing data type by constant value which is zero, and the no response missing data type by impute statistical value.
</pre><p> </p><p>mistake: "not required " should be "not required".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>